java-api-core/src/com/kinvey/java/store/BaseDataStore.java

```diff
@@ -302,6 +331,9 @@ public void pushBlocking() throws IOException {
         return networkData;
     }
 
+    public void getCountBlocking() throws IOException {
```

Comment:
Does this method need to be a public method on the `BaseDataStore`?

Reply:
Actually, I will remove this method, as it is not used anymore.